packages/devtools-reps/src/reps/tests/element-node.js

```diff
@@ -0,0 +1,320 @@
+const React = require("react");
+const { shallow } = require("enzyme");
+const {
+  REPS,
+  getRep,
+} = require("../rep");
+const { MODE } = require("../constants");
+const { ElementNode } = REPS;
+const { getSelectableInInspectorGrips } = require("./test-helpers");
+const stubs = require("../stubs/element-node");
+
+describe("ElementNode - BodyNode", () => {
+  const stub = stubs.get("BodyNode");
+
+  it("selects ElementNode Rep", () => {
+    expect(getRep(stub)).toBe(ElementNode.rep);
+  });
+
+  it("renders with expected text content", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub
+    }));
+
+    expect(renderedComponent.text())
+      .toEqual('<body id="body-id" class="body-class">');
+  });
+
+  it("renders with expected text content on tiny mode", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub,
+      mode: MODE.TINY
+    }));
+
+    expect(renderedComponent.text())
+      .toEqual("body#body-id.body-class");
+  });
+});
+
+describe("ElementNode - DocumentElement", () => {
+  const stub = stubs.get("DocumentElement");
+
+  it("selects ElementNode Rep", () => {
+    expect(getRep(stub)).toBe(ElementNode.rep);
+  });
+
+  it("renders with expected text content", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub
+    }));
+
+    expect(renderedComponent.text()).toEqual('<html dir="ltr" lang="en-US">');
+  });
+
+  it("renders with expected text content in tiny mode", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub,
+      mode: MODE.TINY
+    }));
+
+    expect(renderedComponent.text()).toEqual("html");
+  });
+});
+
+describe("ElementNode - Node", () => {
+  const stub = stubs.get("Node");
+  const grips = getSelectableInInspectorGrips(stub);
+
+  it("has one node grip", () => {
+    expect(grips.length).toEqual(1);
+  });
+
+  it("selects ElementNode Rep", () => {
+    expect(getRep(stub)).toBe(ElementNode.rep);
+  });
+
+  it("renders with expected text content", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub
+    }));
+
+    expect(renderedComponent.text()).toEqual(
+      '<input id="newtab-customize-button" class="bar baz" dir="ltr" ' +
+      'title="Customize your New Tab page" value="foo" type="button">'
+    );
+  });
+
+  it("renders with expected text content in tiny mode", () => {
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stub,
+      mode: MODE.TINY
+    }));
+
+    expect(renderedComponent.text())
+      .toEqual("input#newtab-customize-button.bar.baz");
+  });
+
+  it("renders and inspect icon", () => {
+    const onInspectIconClick = jest.fn();
+    const renderedComponent = shallow(ElementNode.rep({
+      object: stubs.get("Node"),
+      onInspectIconClick
+    }));
+
+    expect(renderedComponent.find(".open-inspector").exists()).toBeTruthy();
+  });
+
+  it("renders and inspect icon", () => {
```

Comment:
I think we can merge this `it` block into the previous one and do the assertion for the click